src/xrGame/ui/UIGameTutorial.h

```diff
@@ -24,6 +24,7 @@ class CUISequencer : public pureFrame, public pureRender, public IInputReceiver
     shared_str m_stop_lua_function;
 
 public:
+    LPCSTR m_name;
```

Comment:
```suggestion
    pcstr m_name;
```

Reply:
Please, make it NOT public

Reply:
If it will made private then we can not got this in script.

![screenshot_2022-05-14T12:11:49](https://user-images.githubusercontent.com/20404541/168419313-4642a4d7-3830-4dcf-be4a-c7415afce98c.jpg)

Reply:
I made this field as private, and replaced it on a method-getter for this filed